spec/spec_helper.rb

```diff
@@ -1,8 +1,16 @@
 require 'rspec'
 require "signable"
+require "support/vcr"
+require "support/webmock"
+require 'dotenv/load'
```

Comment:
I dont believe we should use dotenv in here. It is a odd requirement of the test suite only

Reply:
What alternative do you suggest that doesn't expose the api key in the codebase?

Reply:
you mean require only on the test case?

Reply:
I guess you are using a real key when recording the interactions between the code and the signable api .. but once recorded it is not needed to execute the tests.

so to include the `dotenv` gem seems overkill. When recording/re-recording the http traffic, you could just do: 
```
SIGNABLE_API_KEY='blahblah' bundle exec rspec
```

Reply:
Will that be apparent to the future dev?

Reply:
I will be clear if we document the process of recording VCR stuff.. which is something we should do regularly

Reply:
Done 👍